examples/mobile/UIComponents/components/appbar/index.html

```diff
@@ -54,6 +65,8 @@ <h1>
 			</ul>
 		</div>
 	</div>
+	<script src="js/app.js">
```

Comment:
Need to check the location of the app.js.

Reply:
fixed